requirements/fabric/base.txt

```diff
@@ -4,6 +4,6 @@
 numpy >=1.17.2, <1.25.1
 torch >=1.11.0, <2.1.0
 fsspec[http]>2021.06.0, <2023.5.0
-packaging >=17.1, <=23.0
+packaging >=20.0, <=23.0
```

Comment:
DeepSpeed requires this minimum version

Reply:
```suggestion
packaging >17.0, <=23.0
```

Reply:
Why this suggestion? The minimum is 20

Reply:
But if you don't use Deepspeed your minimum is much lower

Reply:
Sure, but there's no way to specify requirement based on other requirements. 20.0 was released in 2020.

Reply:
ok...